docs/faq.md

```diff
@@ -57,18 +57,6 @@ bundle install --no-deployment --without development --path .bundle
 bundle exec librarian-puppet install --path=shared --clean
 ```
 
-### Q: What's a good approach to merging our-boxen back into my private fork?
```

Comment:
The question suggests it's for upgrading, but I see what you mean - I'll remove this commit and just leave the `--path` fix on its own.